particle-accelerator/src/renderer/views/LoginView.jsx

```diff
@@ -0,0 +1,31 @@
+import { useContext } from 'react'
+import LoginContext from '../utilities/LoginContext'
+import TokenContext from '../utilities/TokenContext'
+import LoginStatus from '../components/LoginStatus'
+import LoginForm from '../components/LoginForm'
+
+/**
+ * LoginView: React Component that provides a login form for a user to login
+ * through to access the ActivatorView.
+ */
+export default function LoginView() {
+  const { setToken, token } = useContext(TokenContext)
+  const { setLoginState, loginState } = useContext(LoginContext)
+
+  return (
+    <>
+      <h2>Particle Accelerator</h2>
+      {/* eslint-disable-next-line react/jsx-no-constructed-context-values */}
```

Comment:
Hmmm, does this memoization break your code?
https://github.com/yannickcr/eslint-plugin-react/blob/master/docs/rules/jsx-no-constructed-context-values.md

Reply:
I decided to ditch the entire global context method and move towards a props-based system for inheriting and modifying states. Although it may make scalability a bit more difficult if the program wanted to grow, it does make everything look much cleaner, and its code that I actually understand.